mava/utils/logger.py

```diff
@@ -281,6 +302,20 @@ def _make_multi_logger(cfg: DictConfig) -> BaseLogger:
     loggers: List[BaseLogger] = []
     unique_token = datetime.now().strftime("%Y%m%d%H%M%S")
 
+    if (
+        cfg.logger.use_neptune
+        and cfg.logger.use_json
+        and cfg.logger.kwargs.upload_json_data
+        and cfg.logger.kwargs.json_path
+    ):
+        raise ValueError(
+            "Cannot upload json data to Neptune when `json_path` is set in the base logger config. "
+            "This is because each subsequent run will create a larger json file which will use "
+            "unnecessary storage. Either set `upload_json_data` if you don't want to upload your "
+            "json data but store a large file locally or remove `json_path` from the base logger "
+            "config."
```

Comment:
Improved the wording a little bit :smile: 

```suggestion
            "unnecessary storage. Either set `upload_json_data: false` if you don't want to "
            "upload your json data but store a large file locally or set `json_path: ~` in "
            "the base logger config."
```